action.yml

```diff
@@ -86,6 +95,33 @@ runs:
         VERSION=$(git tag --list 'firtool-*' | tail -n1)
         echo version=$VERSION >> $GITHUB_OUTPUT
         echo Latest version is: \`$VERSION\` >> $GITHUB_STEP_SUMMARY
+    - name: Decide whether or not to open PR
+      id: open-pr
+      shell: bash
+      run: |
+        VERSION=${{ steps.get-version-latest.outputs.version }}
+        if [[ ${{ steps.get-version-current.outputs.version }} != "$VERSION" ]]; then
+          if [[ -n "${{ inputs.should-create-pr }}" ]]; then
+            if $(downstream/${{ inputs.should-create-pr }} "$VERSION"); then
+              RESULT="0"
+            else
+              RESULT=$?
+            fi
+          else
+            RESULT="0"
+          fi
+          if [[ "$RESULT" -eq "0" ]]; then
+            echo "result=yes" >> $GITHUB_OUTPUT
```

Comment:
Can the PR use a GitHub-documented boolean instead of a `yes`/`no`. No Javascript shenanigans here...